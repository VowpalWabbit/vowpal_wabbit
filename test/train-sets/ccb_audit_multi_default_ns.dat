ccb shared | a
ccb action | b
ccb action |
ccb slot | a
ccb slot | b

