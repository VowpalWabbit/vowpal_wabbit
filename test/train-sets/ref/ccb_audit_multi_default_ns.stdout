[warning] Any duplicate namespace interactions will be removed
You can use --leave_duplicate_interactions to disable this behaviour.
0
	b:163331:1:0@0	a:92594:1:0@0	Constant:116060:1:0@0	a:92594:1:0@0	_ccb_slot_index^index0:68306:1:0@0	b*b:139450:1:0@0	b*a:79627:1:0@0	a*a:2692:1:0@0	b*a:79627:1:0@0	a*a:2692:1:0@0	b*_ccb_slot_index^index0:87147:1:0@0	a*_ccb_slot_index^index0:27108:1:0@0	a*a:2692:1:0@0	a*_ccb_slot_index^index0:27108:1:0@0	b*b*_ccb_slot_index^index0:36380:1:0@0	b*a*_ccb_slot_index^index0:44163:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0	b*a*_ccb_slot_index^index0:44163:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0
0
	Constant:116060:1:0@0	a:92594:1:0@0	a:92594:1:0@0	_ccb_slot_index^index0:68306:1:0@0	a*a:2692:1:0@0	a*a:2692:1:0@0	a*_ccb_slot_index^index0:27108:1:0@0	a*a:2692:1:0@0	a*_ccb_slot_index^index0:27108:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0
0
	b:163331:1:0@0	a:92594:1:0@0	Constant:116060:1:0@0	a:92594:1:0@0	_ccb_slot_index^index0:68306:1:0@0	b*b:139450:1:0@0	b*a:79627:1:0@0	a*a:2692:1:0@0	b*a:79627:1:0@0	a*a:2692:1:0@0	b*_ccb_slot_index^index0:87147:1:0@0	a*_ccb_slot_index^index0:27108:1:0@0	a*a:2692:1:0@0	a*_ccb_slot_index^index0:27108:1:0@0	b*b*_ccb_slot_index^index0:36380:1:0@0	b*a*_ccb_slot_index^index0:44163:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0	b*a*_ccb_slot_index^index0:44163:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0
0
	Constant:116060:1:0@0	a:92594:1:0@0	a:92594:1:0@0	_ccb_slot_index^index0:68306:1:0@0	a*a:2692:1:0@0	a*a:2692:1:0@0	a*_ccb_slot_index^index0:27108:1:0@0	a*a:2692:1:0@0	a*_ccb_slot_index^index0:27108:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0	a*a*_ccb_slot_index^index0:100126:1:0@0
0
	Constant:116060:1:0@0	a:92594:1:0@0	b:163331:1:0@0	_ccb_slot_index^index1:25433:1:0@0	a*a:2692:1:0@0	a*b:204085:1:0@0	a*_ccb_slot_index^index1:65647:1:0@0	b*b:139450:1:0@0	b*_ccb_slot_index^index1:15840:1:0@0	a*a*_ccb_slot_index^index1:61077:1:0@0	a*b*_ccb_slot_index^index1:170294:1:0@0	b*b*_ccb_slot_index^index1:124823:1:0@0
0
	Constant:116060:1:0@0	a:92594:1:0@0	b:163331:1:0@0	_ccb_slot_index^index1:25433:1:0@0	a*a:2692:1:0@0	a*b:204085:1:0@0	a*_ccb_slot_index^index1:65647:1:0@0	b*b:139450:1:0@0	b*_ccb_slot_index^index1:15840:1:0@0	a*a*_ccb_slot_index^index1:61077:1:0@0	a*b*_ccb_slot_index^index1:170294:1:0@0	b*b*_ccb_slot_index^index1:124823:1:0@0
